### ISSUE TITLE
Support for getting mDns name

### ISSUE THREAD

**Issue by guyluz11** [git-elliot/network_tools#6]
Will be cool to get mDns of each IP with the result.

mDns / NSD / DNS-SD / Bonjour.

**Comment by git-elliot** [comment]
Can you please look into this package and see if we can get anything from this package [multicast_dns](https://pub.dev/packages/multicast_dns)

**Comment by guyluz11** [comment]
I didn't manage to find our desired function in this package, so I have opened an issue for that

https://github.com/flutter/flutter/issues/96755

**Comment by guyluz11** [comment]
My bed, `nsd` does not belong here.

**Comment by guyluz11** [comment]
We can get IP from mDNS name so this one will also be good for us

https://github.com/flutter/flutter/issues/97210

**Comment by guyluz11** [comment]
I have created pr for finding mdns on the network, but this feature is limited to finding only srv records that are on the list.
This will not find all mdns devices on the network and is not efficient but this is currently our only option for dart native.
I have added in the code link to the feature request of that in [multicast_dns](https://pub.dev/packages/multicast_dns).

For our new and coming `network_tools_flutter` implementation we will override this method and will call a function that is flutters only and better implemented from [nsd](https://pub.dev/packages/nsd) package or other package.
